app/models/create_expenses_with_receipt.rb

```diff
@@ -0,0 +1,29 @@
+class CreateExpensesWithReceipt
+  def initialize(expense_params)
+    @params = expense_params
+  end
+
+  def create
+    return false unless picture_exists?
+
+    Expense.transaction do
+      create_expense
+      create_receipt
+    end
+    @expense
+  end
```

Comment:
@zamith Should `create` return true/false (depending on whether it was successful or not)? If so, I could I do that here?